### DISCUSSION TITLE
Big latency between non-local network

### DISCUSSION THREAD

**Discussion by deathbull** [jchristn/WatsonTcp#207] [General]
I tried with 2 machine and my local machine, i can run the library good enough but when i run it on non local machine it really has latency i'm sure its about library because i tested WatsonWebsocket it was much better than this.

**Comment by jchristn** [comment]
Can you isolate where in the library you're encountering the latency?  Ie something like .NET Reflector?

**Comment by deathbull** [comment]
I couldn't find the code where the delay happening.

**Comment by jchristn** [comment]
Ok, moving this to a discussion.  If you can pinpoint the issue, please let me know.  I haven't experienced this.
